ruma-events/src/policy/rule.rs

```diff
@@ -55,6 +55,6 @@ impl Display for Recommendation {
 
 impl From<Recommendation> for String {
     fn from(recommendation: Recommendation) -> String {
-        recommendation.to_string()
+        recommendation.into()
```

Comment:
This doesn't work, it will do an endless recursion. I was only talking about `&str` -> `String`, not anything -> String ;)

Reply:
Oops :)